examples/svelte-starter-kit/README.md

```diff
@@ -0,0 +1,38 @@
+# create-svelte
```

Comment:
I think it's important & useful to cleanup the README a bit so it also includes info about Builder. See https://github.com/BuilderIO/builder/tree/main/examples/svelte/sveltekit , you can probably copy/paste most of it for this one

Reply:
It was indeed outdated, just updated based on what it really is :)